app.json

```diff
@@ -38,7 +38,12 @@
       "quantity": 1
     }
   },
-  "addons": [],
+  "addons": [
+    {
+      "plan": "memcachedcloud:30",
+      "as": "MEMCACHED_CLIENT"
+    }
+  ],
```

Comment:
This creates the addon for Heroku for each review app and sets the 3 needed env variables directly with the prefix `MEMCACHED_CLIENT`.

I took the free plan with 30Mb of memory.